.travis.yml

```diff
@@ -41,6 +41,7 @@ before_install:
 
 install:
   - travis_retry composer install --no-interaction --ignore-platform-reqs
+  - php bin/update_hostname_validator.php --check-only
```

Comment:
* move this to the last script command
* make the command to execute only if CHECK_HOSTNAME env var is set. This will set only on php 5.5